classes/Pods.php

```diff
@@ -1675,7 +1659,9 @@ public function field( $name, $single = null, $raw = false ) {
 
 									foreach ( $data as $item_id => $item ) {
 										// $field is 123x123, needs to be _src.123x123
-										$full_field = implode( '.', array_splice( $params->traverse, $key ) );
+										$traverse_fields    = array_splice( $params->traverse, $key );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 1 space but found 4 spaces

<sub>hash: eb5777b2ce805d2ecfca87006cbd33cd</sub>